src/flisp/flisp.h

```diff
@@ -38,10 +38,7 @@ typedef struct _symbol_t {
     // below fields are private
     struct _symbol_t *left;
     struct _symbol_t *right;
-    union {
-        char name[1];
-        void *_pad;    // ensure field aligned to pointer size
-    };
+    char name[] __attribute__((aligned(sizeof(void*))));
```

Comment:
ref https://github.com/JuliaLang/julia/blob/9fe603724a738fe6289f152c0abea5bfea4f58e6/src/julia.h#L52-L62 on the alignment, but I think variable-length arrays in structs might be a gnu extension